tests/js/server/replication/fuzz/replication-fuzz-global.js

```diff
@@ -59,7 +62,12 @@ function ReplicationSuite() {
   };
 
   var collectionChecksum = function(name) {
-    return db._collection(name).checksum(false, true).checksum;
+    if (isCluster) {
+      let csa = db._query('RETURN md5(FOR doc IN @@col SORT doc._key RETURN [doc._key, doc._rev])', {'@col':name}).toArray();
```

Comment:
This will be expensive to run on large collections.
What is the approximate size of data we will run this on?

Reply:
we currently don't have another way to tell collections are identical on both sides in the cluster case. Since this needs to be running on 3.6 as well (and is not triggered via testing.js atm) I think we should leave it that way until we get a better idea.